x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/init_test_bed.tsx

```diff
@@ -1,55 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import React from 'react';
-import { registerTestBed, TestBedConfig } from '@kbn/test-jest-helpers';
-import { docLinksServiceMock } from '@kbn/core/public/mocks';
-
-import '../helpers/global_mocks';
-
-import { licensingMock } from '@kbn/licensing-plugin/public/mocks';
-import { EditPolicy } from '../../../public/application/sections/edit_policy';
-import { KibanaContextProvider } from '../../../public/shared_imports';
-import { AppServicesContext } from '../../../public/types';
-import { createBreadcrumbsMock } from '../../../public/application/services/breadcrumbs.mock';
-import { POLICY_NAME } from './constants';
-
-const getTestBedConfig = (testBedConfigArgs?: Partial<TestBedConfig>): TestBedConfig => {
-  return {
-    memoryRouter: {
-      initialEntries: [`/policies/edit/${POLICY_NAME}`],
-      componentRoutePath: `/policies/edit/:policyName`,
-    },
-    ...testBedConfigArgs,
-  };
-};
-
-const breadcrumbService = createBreadcrumbsMock();
-
-const EditPolicyContainer = ({ appServicesContext, ...rest }: any) => {
```

Comment:
Since we now have a generic `WithAppDependencies` HoC there's no need to define all of these dependencies. I've renamed the file to `init_test_bed.ts` and removed all the code related to that.